src/wamr/funcs.cpp

```diff
@@ -2,6 +2,7 @@
 #include <faabric/util/bytes.h>
 #include <faabric/util/macros.h>
 #include <wamr/native.h>
+#include <wasm/WasmExecutionContext.h>
```

Comment:
Is this include necessary here? It's not used in the file elsewhere.

Reply:
It is, because the `getExecutingCall`/ `getExecutingModule` functions now live in this header rather than `WasmModule.h`

Reply:
You are right I had missed that :+1: